rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/CloudProviderBakeHandler.groovy

```diff
@@ -140,7 +141,21 @@ abstract class CloudProviderBakeHandler {
     def virtualizationSettings = findVirtualizationSettings(region, bakeRequest)
 
     BakeOptions.Selected selectedOptions = new BakeOptions.Selected(baseImage: findBaseImage(bakeRequest))
-    def (imageName, appVersionStr, packagesParameter) = imageNameFactory.deriveImageNameAndAppVersion(bakeRequest, selectedOptions)
+
+    List<String> packageNameList = bakeRequest?.package_name?.tokenize(" ")
```

Comment:
Since `packageNameList` can end up `null`, and the `.collect` down below will return an empty list when applied to it, the `osPackageNames.first()` just inside of `buildImageName()` will likely throw an exception when `package_name` is `null` or an empty string.